Realm/RLMResults.h

```diff
@@ -408,6 +427,174 @@ __attribute__((warn_unused_result));
                                                          NSError *_Nullable error))block
                                       keyPaths:(nullable NSArray<NSString *> *)keyPaths
 __attribute__((warn_unused_result));
+
+#pragma mark - Flexible Sync
+
+/**
+ Creates an RLMSyncSubscription matching the RLMResults' local filter.
+
+ This method opens a write transaction that creates or updates a subscription.
```

Comment:
I'll add this as a note and a link to that API

Reply:
Sorry - how so? Like a hyperlink? Or a note below that says : 
```- note: The update transaction used is `[RLMSyncSubscription update:queue:onComplete:]`.
```